src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultControllerModelBuilder.cs

```diff
@@ -80,6 +85,22 @@ protected virtual ControllerModel CreateControllerModel([NotNull] TypeInfo typeI
             AddRange(controllerModel.Filters, attributes.OfType<IFilter>());
             AddRange(controllerModel.RouteConstraints, attributes.OfType<IRouteConstraintProvider>());
 
+            var enableCors = attributes.OfType<IEnableCorsMetadata>().SingleOrDefault();
```

Comment:
A scenario to consider: Enable cors for the whole app and restrict the policy to certain origins that I trust. (basically as much less work as possible for end user without using attributes maybe)